src/structures/User.js

```diff
@@ -255,6 +262,22 @@ class User extends Base {
     return equal;
   }
 
+  /**
+   * Fetches this user's flags.
+   * @returns {Promise<UserFlags>}
+   */
+  fetchFlags() {
```

Comment:
If the flags are fetched, you don't return a promise.
```suggestion
  async fetchFlags() {
```